client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -258,6 +260,17 @@ public ReaderGroupConfigBuilder retentionType(StreamDataRetention type) {
            return this;
        }
 
+       /**
+        * Set the rollover size for the {@link ReaderGroup} internal stream segment.
+        *
+        * @param rolloverSizeBytes rollover size for the internal stream segment.
+        * @return Reader group config builder.
+        */
+       public ReaderGroupConfigBuilder rolloverSizeBytes(final long rolloverSizeBytes) {
+           this.rolloverSizeBytes = rolloverSizeBytes;
```

Comment:
Unless we require setting a different _`rolloverSizeBytes`_ for each Reader Group Stream, this value can be chosen by the Controller without requiring the user to set it.